apps/server/README.md

```diff
@@ -0,0 +1,18 @@
+# Ajó Savings
+
+server side.
+
+# Project Setup.
+
+1. clone project.
+2. run yarn.
+3. navigate to server directory.
+4. create .env.production and .env.developement files.
+5. copy the env template in .env.example into the files created in step 4 above.
+6. supply the values for your developement and production env.
+7. run yarn dev to start server in developement mode.
+
+# Starting the server in production mode.
+
+1. navigate to server directory ("./apps/server")
+2. run yarn prod
```

Comment:
this info should be at the root dir

Reply:
Alrigt boss. Should I move the README file to root directory and rename it or should I just  add the content of this one to the readme in the root dir?

Reply:
> Alrigt boss. Should I move the README file to root directory and rename it or should I just add the content of this one to the readme in the root dir?

I think he meant that you should add the newly added content to the READMe file in root dir. Also, correct the spelling of 'developement' on lines 10, 12, and 13.